server/lib/src/server/migrations.rs

```diff
@@ -1013,6 +1013,7 @@ impl<'a> QueryServerWriteTransaction<'a> {
         let idm_schema_classes: Vec<EntryInitNew> = vec![
             SCHEMA_CLASS_ACCOUNT.clone().into(),
             SCHEMA_CLASS_ACCOUNT_POLICY.clone().into(),
+            SCHEMA_CLASS_BUILTIN.clone().into(),
```

Comment:
Ah, will do 😄

Reply:
I really should put a big warning there saying "NEVER CHANGE THIS ITS THE WRONG SPOT"

Reply:
So... they can be there, just have to be in the migration as well? Or rename  `idm_schema_classes: Vec<EntryInitNew>` to `idm_schema_classes_dl1`  with a note about future things being migrations?

Reply:
Whene you have it (idm_schema_migrate )? is only run on "first install" of an instance. So an upgrade or re-run won't apply it.

But on a first run *all* the domain migrations are applied too. I'd say only put it into the dl 5 to 6 migration just because that gives testing options with the new test framework we merged (if we choose to use it here).